report_thesis/src/sections/experiment_design/index.tex

```diff
@@ -0,0 +1,8 @@
+\section{Experimental Design}\label{sec:methodology}
+In this section, we outline the methodology used in this study to address the challenges identified in Section~\ref{sec:problem_definition}. Our objective was to identify the most promising machine learning models and preprocessing techniques proposed from the literature, as outlined in Section~\ref{sec:related-work}, for predicting major oxide compositions from \gls{libs} data.
+Then, using this knowledge, develop a pipeline that utilizes the strengths of these models and preprocessing techniques to improve prediction accuracy and robustness of the predictions.
+We first describe the datasets used, including their preparation and the method of splitting for model training. Next, we outline the preprocessing steps and the model selection process, followed by a detailed explanation of the experimental setup and evaluation metrics. Finally, we discuss our validation testing procedures and the approach taken to ensure unbiased final model evaluations.
```

Comment:
This introduction is really weird. It feels out of place & like it describes an entirely different section.
Need to rewrite.

I think this is a leftover from before we made a bunch of changes?